### PR TITLE
Add canBeDeactivated

### PR DESCRIPTION
**Pull request by andreas-junghanns**

### CODE REVIEW

File: docs/2_3_common_states.adoc [hunk 1]
```diff
@@ -479,9 +479,9 @@ There are multiple kinds of events that require a transition to <<EventMode>>:
 a|
 * Activate clocks and respective model partitions in latexmath:[\mathbf{f}_{\mathit{event}}]
 * Deactivate clocks and
-  - Reset respective clocked variables to their previous values: latexmath:[\mathbf{v}_{\mathit{d}} := {}^\bullet\mathbf{v}_{\mathit{d}}]
+  - Reset respective clocked variables to their previous values: latexmath:[\mathbf{v}_{\mathit{k}} := {}^\bullet\mathbf{v}_{\mathit{k}}]
   - Deactivate respective model partitions in latexmath:[\mathbf{f}_{\mathit{event}}]
-|<<fmi3SetClock>>
+|<<fmi3SetClockEM>>
```
Comment (KarlWernersson) [inline docs/2_3_common_states.adoc:484]: Is the EM intentional?
What does it mean?

File: docs/2_3_common_states.adoc [hunk 2]
```diff
@@ -518,10 +518,13 @@ This function can be called for variables with <<causality,`causality == input`>
 Functions <<get-and-set-variable-values,`fmi3Get{VariableType}`>>::
 Getting variables might trigger <<selectiv-computation,computations>>.
 
+[[fmi3SetClockEM,`fmi3SetClock`]]
```
Comment (KarlWernersson) [inline docs/2_3_common_states.adoc:521]: EM here to

**Review by KarlWernersson — COMMENTED** [review]
Looks good.
In addition I added some comments in #1402 that should be decided and might require additional changes.

File: docs/2_3_common_states.adoc (hunk 1 above)
Comment (andreas-junghanns) [reply docs/2_3_common_states.adoc:484]: This is the xref to the fmi3SetClock description for Event Mode (EM).

File: docs/2_3_common_states.adoc (hunk 2 above)
Comment (andreas-junghanns) [reply docs/2_3_common_states.adoc:521]: > EM here to

Yes, exactly - as intended.

File: docs/2_3_common_states.adoc (hunk 1 above)
Comment (KarlWernersson) [reply docs/2_3_common_states.adoc:484]: OK thanks

**Comment by andreas-junghanns** [comment]
@KarlWernersson : the other issues are still open on top of this issue here. I would like to merge this and have changes/clarifications added through the other issue. If you think #1402 is not sufficiently addressed by this PR, we (you) should unlink #1402 from this PR here and still approve this PR so we can merge. Please. Soon. SC meeting comming up, I want to close as many issues and PRs as possible if they are clear enough to be an improvement.

**Review by KarlWernersson — APPROVED** [review]
I am not convinced that this is a good solution but people seems to want it so let put it there for now. We can always remove it in beta if implementation points to further problems.

**Pull request merged**